magento/magento2
# fix for #38585: Signed primary columns changed value

### Description (*)
changed "unsigned" attribute for "value_id" column for tables from original list (This is applicable to the following tables:... from author post) + catalog_category_entity_datetime, customer_address_entity_datetime


### Related Pull Requests


### Fixed Issues (if relevant)

1. Fixes magento/magento2#38585

### Manual testing scenarios (*)

1. mysql -u root -p
2. use (your magento2 db name)
3. desc catalog_product_entity_text (or any table from the list)
4. check "value_id" type, should be unsigned
<img width="567" alt="image" src="https://github.com/user-attachments/assets/223eb2cc-40b8-464b-9c15-d7d8e666d4dc">


### Questions or comments


### Contribution checklist (*)
